python/setup.py

```diff
@@ -290,6 +290,7 @@ def get_packages(self):
         "frozenlist",
         "requests",
         "virtualenv",  # For pip runtime env.
+        "Jinja2 >= 3.0.3",
```

Comment:
could we only take this dep in ray[default]?

Reply:
For sure!